src/v/cloud_storage/segment_meta_cstore.cc

```diff
@@ -267,22 +268,20 @@ class column_store
     /// Add element to the store. The operation is transactional.
```

Comment:
nit: not sure we want to leave this comment

Reply:
Yes, but it still guarantees that the element is added fully or not added at all.